code/game/objects/items/circuitboards/machine_circuitboards.dm

```diff
@@ -271,9 +271,9 @@
 		/obj/item/stack/cable_coil = 1,
 		/obj/item/stack/sheet/glass = 1)
 
-/obj/item/circuitboard/machine/thermomachine/multitool_act(mob/living/user, obj/item/multitool/I)
+/obj/item/circuitboard/machine/thermomachine/multitool_act(mob/living/user, obj/item/multitool/multitool)
 	. = ..()
-	if (istype(I))
+	if (istype(multitool))
```

Comment:
I don't even understand why this check is here, CC @Ghilker is this yours?

Reply:
yeah i added it back then because i saw other multitool_act using this behaviour so i put it in, didn't know if was useful so i left it in

Reply:
i don't see anything else using it... so i'll just remove it